### ISSUE TITLE
invalid hook call

### ISSUE THREAD

**Issue by dereknelson** [appmotion/apollo-augmented-hooks#2]
hey, love the idea of this package! 

right now when using the package i get the error:
```
Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons...
```
this is because in the package.json there is a dependency on react 17, instead of a peer dependency. 

additionally, the [`queryAst` pattern (eg, always wrapping the query argument with `gql`) ](https://github.com/appmotion/apollo-augmented-hooks/blob/master/src/useQuery.js#L36)makes it difficult to migrate our queries from the normal apollo client that we had prefixed the template literals with gql one at a time. would love to help you come up with a solution that takes either
```
gql(`queryStuff`)
or just
`queryStuff`
```
as an argument.

**Comment by mindnektar** [comment]
Thanks a lot for your feedback! I've just released v1.8.0 in which both of these issues are addressed.

**Comment by dereknelson** [comment]
@mindnektar awesome, thanks for the quick fixes! i'll be using this library a lot, looking forward to working on more improvements with you!
